tests/unit/test_collection.py

```diff
@@ -77,7 +77,7 @@
   "name": "FILES.json",
   "ftype": "file",
   "chksum_type": "sha256",
-  "chksum_sha256": "dc90402feea54f479780e067cba748559cb01bff52e6724a15264b9a55e8f000",
+  "chksum_sha256": "0c2e9514f54aaeda0605d8c9a60e71cab3e577113b58ecc65b839d2d1b86c216",
```

Comment:
_<general comment on existing code, not a requested change...>_

Updating the tests to calculate this on the fly might be useful in the future. I didn't do it in an attempt to stay focused on the feature. Maybe it's better as is for explicitness, I'm not sure. It is a tiny bit annoying to update though.

Reply:
Haven't had to update this much so should be ok, and wasn't a bad update